<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1031</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="pageLogin">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>140</y>
        <width>231</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="loLogin">
       <item>
        <widget class="QLabel" name="labelStudentLogin">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Login&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblStudentUserName">
           <property name="text">
            <string>UserName</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputStudentLoginUsername"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="loLoginPassword">
         <item>
          <widget class="QLabel" name="lblStudentPassword">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputStudentLoginPassword"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnLogin">
         <property name="text">
          <string>Log In</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelStudentLoginBadpass">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:7pt; font-weight:600;&quot;&gt;Incorrect Password &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:7pt; font-weight:600;&quot;&gt;Please try again&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pageDbLogin">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>281</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="loDbLogin">
       <item>
        <widget class="QLabel" name="labelDatabaseLoginTitle">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Database Login&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="loDbLoginUsername">
         <item>
          <widget class="QLabel" name="lblDbloginUserName">
           <property name="text">
            <string>Database UserName</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputDbloginUsername"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="loDbLoginPassword">
         <item>
          <widget class="QLabel" name="lblDbloginPassword">
           <property name="text">
            <string>Database Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputDbloginPassword"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnDbloginLogin">
         <property name="text">
          <string>Log Into Database</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnDbloginBack">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Student Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDbLoginGotoFacultyLogin">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>220</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Faculty Login</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageFacultyLogin">
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>231</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="loFacultyLogin">
       <item>
        <widget class="QLabel" name="labelStudentLogin_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Faculty Login&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblFacultyName">
           <property name="text">
            <string>UserName</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputFacultyUsername"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="loLoginPassword_2">
         <item>
          <widget class="QLabel" name="lblFacultyPassword">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputFacultyPassword"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnFacultyLogin">
         <property name="text">
          <string>Faculty Log In</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelFacultyLoginBadpass">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:7pt; font-weight:600;&quot;&gt;Incorrect Password &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:7pt; font-weight:600;&quot;&gt;Please try again&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnFacultyLoginGotoStudentLogin">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>270</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Student Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFacultyLoginGotoDBLogin">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Database Login</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageDbAdmin">
     <widget class="QWidget" name="widgetDatabase" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>771</width>
        <height>521</height>
       </rect>
      </property>
      <widget class="QPushButton" name="btnDbAdminDropTable">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>340</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Drop All Tables</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDbAdminLogout">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>430</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Log Out</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDbAdminAddTables">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>370</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add New Tables</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDbAdminSeedTables">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>400</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Seed Tables</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textDbAdminConsole">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>10</y>
         <width>751</width>
         <height>321</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="pageStudent">
     <widget class="QTabWidget" name="tabStudent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>1031</width>
        <height>321</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabRegistration">
       <attribute name="title">
        <string>Current Registration</string>
       </attribute>
       <widget class="QTableWidget" name="tableCurrentRegistration">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1001</width>
          <height>281</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Semester</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Course</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Registered</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Credits</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Grade</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tabRegister">
       <attribute name="title">
        <string>Register for Classes</string>
       </attribute>
       <widget class="QTableWidget" name="tableRegister">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1001</width>
          <height>281</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Semester</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Course</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Credits</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Description</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tabCourses">
       <attribute name="title">
        <string>Course History</string>
       </attribute>
       <widget class="QTableWidget" name="tableCourseHistory">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1011</width>
          <height>281</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Semester</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Course</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Status</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Registered</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Completed</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Credits</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Grade</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>371</width>
        <height>194</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="loStudentInfo">
       <item>
        <layout class="QHBoxLayout" name="loStudentId">
         <item>
          <widget class="QLabel" name="labelStudentId">
           <property name="minimumSize">
            <size>
             <width>107</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Student ID: </string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputStudentId">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="loStudentFirstName">
         <item>
          <widget class="QLabel" name="labelStudentFirstName">
           <property name="minimumSize">
            <size>
             <width>107</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>First Name: </string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputStudentFirstName">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="loStudentLastName">
         <item>
          <widget class="QLabel" name="labelStudentLastName">
           <property name="minimumSize">
            <size>
             <width>107</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Last Name: </string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputStudentLastName">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="loStudentEmail">
         <item>
          <widget class="QLabel" name="labelStudentEmail">
           <property name="minimumSize">
            <size>
             <width>107</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Email Address: </string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputStudentEmail">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="loStudentGPA">
         <item>
          <widget class="QLabel" name="labelStudentGPA">
           <property name="minimumSize">
            <size>
             <width>107</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>GPA: </string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputStudentGPA">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnLogOut">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>550</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Log Out</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageFaculty">
     <widget class="QPushButton" name="btnFacultyLogout">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>320</y>
        <width>291</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Log Out</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>371</width>
        <height>194</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="loFacultyInfo">
       <item>
        <layout class="QHBoxLayout" name="loFacultyId">
         <item>
          <widget class="QLabel" name="labelFacultyId">
           <property name="minimumSize">
            <size>
             <width>107</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Faculty ID</string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputFacultyId">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="loFacultyFirstName">
         <item>
          <widget class="QLabel" name="labelFacultyFirstName">
           <property name="minimumSize">
            <size>
             <width>107</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>First Name: </string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputFacultyFirstName">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="loFacultyLastName">
         <item>
          <widget class="QLabel" name="labelFacultyLastName">
           <property name="minimumSize">
            <size>
             <width>107</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Last Name: </string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputFacultyLastName">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="loFacultyEmail">
         <item>
          <widget class="QLabel" name="labelFacultyEmail">
           <property name="minimumSize">
            <size>
             <width>107</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Email Address: </string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputFacultyEmail">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabFaculty">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>371</width>
        <height>111</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Students</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Semesters</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Course List</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="pageLogOut">
     <widget class="QTextEdit" name="textLogoutConsole">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>761</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLogoutBacktologin">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>340</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back to login</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
